superset-frontend/src/SqlLab/components/TableElement.tsx

```diff
@@ -247,41 +256,17 @@ const TableElement = props => {
     return metadata;
   };
 
-  const collapseExpandIcon = () => (
-    <IconTooltip
-      style={{
-        position: 'fixed',
-        right: '16px',
-        left: 'auto',
-        fontSize: '12px',
-        transform: 'rotate(90deg)',
-        display: 'flex',
-        alignItems: 'center',
-      }}
-      aria-label="Collapse"
-      tooltip={t(`${isActive ? 'Collapse' : 'Expand'} table preview`)}
-    >
-      <Icons.RightOutlined
-        iconSize="s"
-        style={isActive ? { transform: 'rotateY(180deg)' } : null}
-      />
-    </IconTooltip>
-  );
-
   return (
     <Collapse.Panel
       {...props}
+      key={key}
```

Comment:
What do you get when console.log key? I am looking at the old component and it looks really weird. 
![Screen Shot 2021-06-10 at 11 12 23 AM](https://user-images.githubusercontent.com/48933336/121550603-e06a8f00-c9dc-11eb-9a47-afd2f003e87e.png)

but key is specifically being put in. 
https://github.com/apache/superset/blob/master/superset-frontend/src/SqlLab/components/SqlEditorLeftBar.jsx#L191

Also, where is this panelKey thing coming from!

Reply:
I pulled your branch and console logged key: 
![Screen Shot 2021-06-10 at 11 24 42 AM](https://user-images.githubusercontent.com/48933336/121552548-8d91d700-c9de-11eb-9d70-cf76f9536d55.png)